app/models/endpoint.rb

```diff
@@ -0,0 +1,25 @@
+class Endpoint < ActiveRecord::Base
+  belongs_to :resource, :polymorphic => true
+
+  default_value_for :verify_ssl, OpenSSL::SSL::VERIFY_PEER
+  validates :verify_ssl, :inclusion => {:in => [OpenSSL::SSL::VERIFY_NONE, OpenSSL::SSL::VERIFY_PEER]}
+
+  def verify_ssl=(val)
+    val = resolve_verify_ssl_value(val)
+    super
+  end
+
+  def verify_ssl?
+    verify_ssl != OpenSSL::SSL::VERIFY_NONE
+  end
```

Comment:
@DNNX so the trick is that we get a `verify_ssl?` for free from Rails.  And, without overriding the method, it will return `true` as long as the `verify_ssl` attribute has a value.  But, there's a semantic meaning to `verify_ssl?` beyond just "does `verify_ssl` have a value?"

For that reason, I think we need to keep `verify_ssl?` as a method to give more meaning to the predicate method even if it's not used yet.

Also, I should point out that we have more work in after this PR to coalesce the meanings of `verify_ssl` (the attribute) and `ExtManagementSystem.security_protocol`; both of which are now being used to indicate some information about SSL support.